src/file/format/ron.rs

```diff
@@ -0,0 +1,74 @@
+use std::collections::HashMap;
+use std::error::Error;
+
+use ron;
+
+use crate::value::{Value, ValueKind};
+
+pub fn parse(
+    uri: Option<&String>,
+    text: &str,
+) -> Result<HashMap<String, Value>, Box<dyn Error + Send + Sync>> {
+    let value = from_ron_value(uri, &ron::from_str(text)?)?;
+    match value.kind {
+        ValueKind::Table(map) => Ok(map),
+
+        _ => Ok(HashMap::new()),
+    }
+}
+
+fn from_ron_value(
+    uri: Option<&String>,
+    value: &ron::Value,
+) -> Result<Value, Box<dyn Error + Send + Sync>> {
+    Ok(match *value {
```

Comment:
Can you please restructure this function implementation to be a bit more one step at a time? :laughing: 

* First of all, you could `use ron::Value` to eliminate the full qualification path (`run::Value::Bool` -> `Value::Bool`).
* Please do use variables (not `Ok(everything)`, but `let v = match { /*...*/ }; Ok(v)`.

Reply:
Ah, you could even let the `match` return the `kind` only and have `Ok(Value::new(uri, kind))` at the end of the function!

Reply:
I've modified the `match` to return the correct `Kind` as asked.

For the qualification, I'd rather keep the `ron::` prefix as to differentiate it from `config`'s own `Value` type.

Reply:
Agreed.